constants.py

```diff
@@ -5,6 +5,16 @@
 import os
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+CRAWLING_INSTRUMENTS = {
```

Comment:
Why should we need this variable?

Reply:
Since the instruments we can trade may not as many as instruments we can crawl, I separate it from TRADING_INSTRUMENTS.
Also, separated them from crawlers to make it more clear to adjust what instruments we can crawl.